src/VisualStudio/Core/Test.Next/Services/ServiceHubServicesTests.cs

```diff
@@ -478,6 +479,21 @@ static async Task AssertSourceGeneratedDocumentsAreSame(Project localProject, Pr
         private static SourceText CreateText(string content, Encoding encoding = null, SourceHashAlgorithm checksumAlgorithm = SourceHashAlgorithm.Sha1)
             => SourceText.From(content, encoding ?? Encoding.UTF8, checksumAlgorithm);
 
+        private static SourceText CreateStreamText(string content, bool useBOM, bool useMemoryStream)
```

Comment:
note: the memory-stream arg didn't change any test results.  but i liked it for completeness as SourceText uses different codepaths for creating itself from a byte[] versus a Stream.